client/layout/hosting-dashboard/style.scss

```diff
@@ -331,3 +331,7 @@
 		}
 	}
 }
+
+.is-full-width-layout .hosting-dashboard-layout__body-notices {
```

Comment:
When the content takes 100% width, not max 1500px, we want the banner to be 100% as well.